src/main/java/gyro/aws/codebuild/WebhookResource.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.codebuild;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import gyro.aws.AwsResource;
+import gyro.aws.Copyable;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Id;
+import gyro.core.resource.Output;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Regex;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidStrings;
+import software.amazon.awssdk.services.codebuild.CodeBuildClient;
+import software.amazon.awssdk.services.codebuild.model.BatchGetProjectsResponse;
+import software.amazon.awssdk.services.codebuild.model.ResourceNotFoundException;
+import software.amazon.awssdk.services.codebuild.model.Webhook;
+import software.amazon.awssdk.services.codebuild.model.WebhookFilter;
+
+@Type("webhook")
+public class WebhookResource extends AwsResource implements Copyable<BatchGetProjectsResponse> {
```

Comment:
The model being passed to should be the model gyro is representing
Here gyro is representing `webhook` then the corresponding model from aws should be that and not some response. The model is primarily determined by the resource and not the finder and what is easier for the finder to return to.

```suggestion
public class WebhookResource extends AwsResource implements Copyable<Webhook> {
```